docs/requirements-tutorials.txt

```diff
@@ -3,4 +3,4 @@ deep-phonemizer
 boto3
 pandas
 librosa
-numpy<=1.20
```

Comment:
Since PyTorch requires `>=1.11`, let's just do `numpy >= 1.21` and see if the `build_doc` job succeeds.